react/week1/react-props/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
```

Comment:
If you run your app, you can see warnings in logs:

```
  Line 1:17:  'Component' is defined but never used  no-unused-vars
  Line 2:8:   'logo' is defined but never used       no-unused-vars
  Line 6:8:   'Header' is defined but never used     no-unused-vars
  Line 7:8:   'Items' is defined but never used      no-unused-vars
```

It's better to fix them

Reply:
Hi @Hatko,

Thank you for the feedback :)
I would note all your points and will implement while writing code going forward. 
I think I forgot to remove the logo variable
Actually we had to write both class component & functional component in App.js file so I left the comments & also above import defined variables.